lib/framework/action.dart

```diff
@@ -257,16 +257,16 @@ class ReceiveIntentAction extends EnsembleAction {
     Invokable? initiator,
     this.options,
     this.id,
-    this.onSuccess,
+    this.onReceive,
     this.onError,
   }) : super(initiator: initiator);
   final Map<String, dynamic>? options;
   String? id;
-  EnsembleAction? onSuccess;
+  EnsembleAction? onReceive;
   EnsembleAction? onError;
 
-  EnsembleAction? getOnSuccess(DataContext dataContext) =>
-      dataContext.eval(onSuccess);
+  EnsembleAction? getOnReceive(DataContext dataContext) =>
```

Comment:
this comment is global across all yuor changes. You don't need to eval the action. They will be eval upon execution.